test/e2e/framework/rc/rc_utils.go

```diff
@@ -26,6 +26,8 @@ import (
 	clientset "k8s.io/client-go/kubernetes"
 	scaleclient "k8s.io/client-go/scale"
 	api "k8s.io/kubernetes/pkg/apis/core"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2ekubectl "k8s.io/kubernetes/test/e2e/framework/kubectl"
 	testutils "k8s.io/kubernetes/test/utils"
 )
 
```

Comment:
We need to rename `RcByNameContainer` for solving the following lint error:

`test/e2e/framework/rc/rc_utils.go:35:6: func name will be used as rc.RcByNameContainer by other packages, and that stutters; consider calling this ByNameContainer`

Reply:
Thanks @oomichi 
Updated.